### PR TITLE
chore:adding update for last week

### PR DESCRIPTION
**Pull request by treagan82**

### CODE REVIEW

File: _posts/2018-04-04-the-one-with-the-meeting.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+layout: post
+title:  "The one with the meeting"
+date:   2018-04-04 1147:00 -0500
+categories: OpenMTR Updates
+tags: Updates
+---
+
+This week we got the pleasure of meeting up with our clients at KUB to give them a brief presentation of OpenMTR.  
+Our hard work this semester has really paid off because they were super impressed!  We received some great
+suggestions for things they like to hear for our final presentation, and we discussed some of the features we have
```
Comment (freudianslip99) [inline _posts/2018-04-04-the-one-with-the-meeting.md]: would like to hear for our final presentation.

File: _posts/2018-04-04-the-one-with-the-meeting.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+planned for implementation that we were able to get some great feedback on and make appropriate changes before we
```
Comment (freudianslip99) [inline _posts/2018-04-04-the-one-with-the-meeting.md]: We discussed some of the features we want to have implemented in our final product.

File: _posts/2018-04-04-the-one-with-the-meeting.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Bryan – This week Machine Learning is at the top of the priority list.  This was a stretch goal we had for ourselves,
+but Matt and Bryan are super confident this can be put in place before we complete the project.  As a secondary
+function Bryan will be looking at cropping photos to make them easier for the program to read.
+
+Alan – Those flowcharts aren’t going to finish themselves!  Alan will be creating new flowcharts for our exit
+documentation this week.  He will also be assisting Todd on web layout adjustments this week.
+
+Todd –  Will be making the last few adjustments to the layouts on the website.  There were also a few adjustments
+suggested during the last meeting that were suggested that he is going to try and squeeze in before we wrap up.
```
Comment (freudianslip99) [inline _posts/2018-04-04-the-one-with-the-meeting.md]: There were also a few suggestion put forth, in our last meeting, that he is going to squeeze in before we wrap up.

File: _posts/2018-04-04-the-one-with-the-meeting.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+documentation this week, making sure no member of the team is drowning in the waters of knowledge.
+
+Nick – Nick has been working on the Demo for weeks, and he has made some fantastic steps in making it something
+special.  This week he will start implementing the finishing touches on this aspect of the project.
+
+Bryan – This week Machine Learning is at the top of the priority list.  This was a stretch goal we had for ourselves,
+but Matt and Bryan are super confident this can be put in place before we complete the project.  As a secondary
+function Bryan will be looking at cropping photos to make them easier for the program to read.
+
+Alan – Those flowcharts aren’t going to finish themselves!  Alan will be creating new flowcharts for our exit
```
Comment (freudianslip99) [inline _posts/2018-04-04-the-one-with-the-meeting.md]: Alan will be working on the final project paper this week.  The paper will be a walk through of the life cycle of OpenMTR.

**Comment by treagan82** [comment]
changes updated.

**Pull request merged**